packages/client/scripts/postBuildFixes.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -e
+
+# Presentational variables and functions declaration.
+
+BLUE="\033[0;34m"
+GREEN="\033[0;32m"
+YELLOW="\033[0;33m"
+RED="\033[0;31m"
+NOCOLOR="\033[0m"
+DIM="\033[2m"
+
+blue() {
+    printf "${BLUE}$1${NOCOLOR}"
+}
+green() {
+    printf "${GREEN}$1${NOCOLOR}"
+}
+yellow() {
+    printf "${YELLOW}$1${NOCOLOR}"
+}
+red() {
+    printf "${RED}$1${NOCOLOR}"
+}
+dim() {
+    printf "${DIM}$1${NOCOLOR}"
+}
+
+post_build_fixes() {
+    blue "[Post Build Fixes]"
+    if [ -f ./dist/esm/bin/cli.js ];
+    then
+        echo "Adding ./dist/cjs/package.json"
+        rm -f ./dist/cjs/package.json
+        cat <<EOT >> ./dist/cjs/package.json
```

Comment:
Why do we need to add a cjs package.json if we're removing cjs support in the client?

Reply:
I just hadn't gone this far yet.  Will remove this entire section of the build script if everyone is good with dropping a CJS build.